module-config-template.yaml

```
@@ -1,8 +1,38 @@
 name: kyma-project.io/module/keda
-channel: {{.Channel}}
 version: {{.Version}}
```

I would suggest being consistent with other fields

```suggestion
version:
```